cloud/blockstore/tools/csi_driver/internal/driver/node.go

```diff
@@ -687,17 +687,62 @@ func logVolume(volumeId string, format string, v ...any) {
 	log.Printf("[v=%s]: %s", volumeId, msg)
 }
 
+func (s *nodeService) doesVolumeExist(
+	ctx context.Context,
+	volumeId string) bool {
+
+	describeVolumeRequest := &nbsapi.TDescribeVolumeRequest{
+		DiskId: volumeId,
+	}
+
+	_, err := s.nbsClient.DescribeVolume(ctx, describeVolumeRequest)
```

Comment:
better check specific error for disk not exist and return error if there is unrelated error

there is function `IsDiskNotFoundError` here that might help https://github.com/ydb-platform/nbs/blob/users/myagkov/issue-1588-fix-csi-sanity-tests/cloud/disk_manager/internal/pkg/clients/nbs/client.go#L358

Reply:
Done

1. I moved IsDiskNotFoundError implementation to cloud/blockstore/public/sdk/go/client/client.go as I don't want to spread "hacky" implementation of this function in different modules.
2. cloud/disk_manager/internal/pkg/clients/nbs/client.go contains wrapper for IsDiskNotFoundError as I don't want to expose import of cloud/blockstore/public/sdk/go/client/client.go in other disk manager modules(e.g. cloud/disk_manager/internal/pkg/services/pools/rebase_overlay_disk_task.go)